recipe/0001-Build-for-all-CUDA-11-arches.patch

```diff
@@ -0,0 +1,34 @@
+From 18cfb7e83418cd2d14298c5d21db56456f4a7f70 Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Thu, 14 Dec 2023 18:31:22 +0000
+Subject: [PATCH] Build for all CUDA 11 arches
+
+---
+ setup.py | 9 ++++++++-
+ 1 file changed, 8 insertions(+), 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index f5e38a4..debad4f 100644
+--- a/setup.py
++++ b/setup.py
+@@ -78,9 +78,16 @@ def compile_cuda_module(host_args):
+     arch_flags = (
+         "-arch=sm_37 "
+         "-gencode=arch=compute_37,code=sm_37 "
++        "-gencode=arch=compute_60,code=sm_60 "
```

Comment:
@jakirkham This is the first time I manually modify an `nvcc` call. Could you (or someone else knowledgable) give this a brief review?